### PR TITLE
Create PDAL arrow

### PR DESCRIPTION
**Pull request by willdunklin**
Created PDAL arrow for point cloud output. Implemented per the example in [TeleSculptor](https://github.com/Kitware/TeleSculptor/blob/master/maptk/write_pdal.h).

### CODE REVIEW

File: arrows/pdal/pdal_config.h.in [hunk 1]
```diff
@@ -0,0 +1,21 @@
+// This file is part of TeleSculptor, and is distributed under the
```
Comment (chetnieter) [inline arrows/pdal/pdal_config.h.in:1]: This should be changed to the KWIVER header

File: arrows/pdal/pointcloud_io.cxx [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+
+  writer->setInput(reader);
+  writer->setOptions(options);
+  writer->prepare(table);
+  writer->execute(table);
+
+#else
+
+  throw vital::file_write_exception(filename,
+                                    "TeleSculptor was not compiled with PDAL, "
```
Comment (chetnieter) [inline arrows/pdal/pointcloud_io.cxx:144]: TeleSculptor -> KWIVER

**Review by chetnieter — CHANGES_REQUESTED** [review]
Looks good. A few spots where you missed changing TeleSculptor to KWIVER. Why is the geo_origin.txt file empty?

**Comment by chetnieter** [comment]
I noticed another weird thing. The `pdal:pointcloud_io.save_geo_origin` test is behaving strangely on my linux box. Sometimes passes and some times fails with different exceptions being thrown by PDAL.

**Comment by chetnieter** [comment]
jenkins test this please

File: arrows/pdal/pointcloud_io.cxx (hunk 2 above)
Comment (willdunklin) [reply arrows/pdal/pointcloud_io.cxx:144]: geo_origin.txt is the direct output of running the pipeline on [this data](https://data.kitware.com/#item/5ef11b419014a6d84ed53971). It being empty still produces seemingly satisfactory output.

Comment (chetnieter) [reply arrows/pdal/pointcloud_io.cxx:144]: I confirmed that adding data to this file fixes the random test failures for me. I used 35.92196902920767 -78.99921072631537 100.3232432432423. Just a random off of google maps.

**Review by chetnieter — APPROVED** [review]

**Pull request merged**